internal/api/v1alpha1/access_config.go

```diff
@@ -29,6 +29,12 @@ type AccessConfig struct {
 	//
 	// +kubebuilder:default:="24h"
 	MaxDuration string `json:"maxDuration"`
+
+	// AccessCommand is used to describe to the user how they can make use of their temporary access
```

Comment:
You should describe here specifically what data this temlate is run against, so that the user understands what options are available.